policyengine_us/variables/gov/states/ri/tax/income/adjusted_gross_income/ri_child_tax_rebate.py

```diff
@@ -0,0 +1,38 @@
+from policyengine_us.model_api import *
+
+
+class ri_child_tax_rebate(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island Child Tax Rebate"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-08/H7123Aaa_CTR_0.pdf"
+    defined_for = StateCode.RI
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("adjusted_gross_income", period)
+        filing_status = tax_unit("filing_status", period)
+        child_count = tax_unit("eitc_child_count", period)
```

Comment:
the law seems to specify 18 years of age and not eitc child count - did I miss another source?